staff/urls.py

```diff
@@ -17,6 +17,7 @@
     path('basics/batches/<int:batch_id>/sections/', section.ListView.as_view(), name='section_list'),
     path('basics/batches/<int:batch_id>/sections/<int:section_id>/', section.DetailView.as_view(), name='section_detail'),
     path('coupons/', coupon.ListView.as_view(), name='coupon_list'),
+    path('coupons/csv-upload/', coupon.CsvUploadView.as_view(), name='coupon_csv_upload'),
```

Comment:
did some reading online - wonder if it makes sense to just POST to `coupons/`. but then we'll still need another route for rendering the form. alternatively we rename this to be `coupons/new/batch/` or `coupons/new_many/` for consistency with the other routes?

https://www.codementor.io/blog/batch-endpoints-6olbjay1hd
https://developer.zendesk.com/api-reference/ticketing/users/users/#create-or-update-many-users
https://stackoverflow.com/questions/28596688/rest-api-bulk-create-or-update-in-single-request
https://stackoverflow.com/questions/28638071/update-an-entire-resource-collection-in-a-rest-way